packages/cli/src/util/input/edit-project-settings.ts

```diff
@@ -83,11 +83,25 @@ export async function editProjectSettings(
     return settings;
   }
 
+  const styledFramework = (frameworkName: string) => {
+    const frameworkStyle = {
+      text: frameworkName,
+      color: chalk.blue,
+    };
+
+    if (frameworkName === 'Hono') {
```

Comment:
Let's ship it and ITG this when there is more, would be great to keep this in the framework config, or maintain another framework config for style data. But love it